company_linkage/sql/ai_company_papers.sql

```diff
@@ -0,0 +1,34 @@
+WITH unnested_rors AS (
```

Comment:
@rggelles this query shows how the new `org_name_matches` table is used. One mystery is that it returns only `89,643 rows`, as opposed to the 130,154 in `staging_ai_companies_visualization.ai_company_papers`. A difference between the old and new methods is that the [new method](https://github.com/georgetown-cset/parat/blob/dee2515a233012be30030de7aa1c9518b516f8c4/company_linkage/sql/org_name_cross_regex.sql) only searches rows where the ror id is null. Do you see any other differences that might account for this?

Reply:
I implemented the fix to regex case sensitivity as discussed last week but this issue persists in a less extreme form - about to post a follow-up below